agent/config/config.go

```diff
@@ -315,6 +315,9 @@ func environmentConfig() (Config, error) {
 		seelog.Debugf("Setting instance attribute %v: %v", attributeKey, attributeValue)
 	}
 
+	containerMetadataEnabled := utils.ParseBool(os.Getenv("ECS_ENABLE_CONTAINER_METADATA"), false)
```

Comment:
For this and the "needs a test" comments below for the config package, I have added tests to check for my new configuration variables.